src/lib/getWikipediaInfo.ts

```diff
@@ -154,21 +190,24 @@ export async function getWikipediaInfo(
   const title = page.title;
   const pageId = page.pageid;
 
-  const isPerson = await isWikipediaPerson({ title, pageId });
-  if (!isPerson) {
+  const isPerson = isWikipediaPerson({ title, pageId });
+  const isDisambiguation = isDisambiguationPage({ pageId });
+
+  const ret: Partial<WikipediaData> = {};
```

Comment:
What’s a ret?

Reply:
@wholesomedev lol, I wanted to use `result` but it's already used in function parameters. So I used `ret` for `return`.